### PR TITLE
Additional option --fully-translated for pull command

### PR DESCRIPTION
**Pull request by jeeyi**
This option is used to prevent partially translated localized files. When this option is specified, it will check the number of untranslated strings for the locale. If there are untranslated strings, the localized file is not generated. 

With this option, localized files are generated for partially translated locales (child locales that inherit translations from the parent locale) only if their parent locales are fully translated.

### CODE REVIEW

File: cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java [hunk 1]
```diff
@@ -98,13 +103,23 @@
...
+
     @Override
     public void execute() throws CommandException {
 
         consoleWriter.newLine().a("Pull localized asset from repository: ").fg(Color.CYAN).a(repositoryParam).println(2);
 
         repository = commandHelper.findRepositoryByName(repositoryParam);
 
+        if (onlyIfFullyTranslated) {
+            checkFullyTranslated(repository);
```
Comment (aurambaj) [inline cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java]: rename to `initLocaleFullyTranslatedMap`?

File: cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java [hunk 2]
```diff
@@ -164,8 +179,23 @@ void generateLocalizedFilesWithLocaleMaping(Repository repository, FileMatch sou
     }
 
     void generateLocalizedFile(Repository repository, FileMatch sourceFileMatch, List<String> filterOptions, String outputBcp47tag, RepositoryLocale repositoryLocale) throws CommandException {
-        LocalizedAssetBody localizedAsset = getLocalizedAsset(repository, sourceFileMatch, repositoryLocale, outputBcp47tag, filterOptions);
-        writeLocalizedAssetToTargetDirectory(localizedAsset, sourceFileMatch);
+        boolean skip = false;
+
+        if (onlyIfFullyTranslated) {
```
Comment (aurambaj) [inline cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java]: move that block to a function?

File: cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java (hunk 1 above)
Comment (jeeyi) [reply cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java]: Done

File: cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java (hunk 2 above)
Comment (jeeyi) [reply cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java]: Done

**Pull request merged**